src/storage-types/src/errors.rs

```diff
@@ -429,6 +468,8 @@ mod columnation {
         string_region: <String as Columnation>::InnerRegion,
         /// Region for storing u8 vectors.
         u8_region: <Vec<u8> as Columnation>::InnerRegion,
+        /// Region for storing bools.
+        bool_region: <bool as Columnation>::InnerRegion,
```

Comment:
You don't need to store `bool` in a region since it won't contain any heap-allocated data!

Reply:
Ah, yes. I got confused by the existence of the `u8` region but that is `Vec<u8>` not just `u8`. Removed!